packages/angular_devkit/build_angular/src/webpack/plugins/karma/karma-debug.html

```diff
@@ -4,7 +4,7 @@
 but its purpose is to be loaded in the main frame (not within an iframe),
 just for immediate execution, without reporting to Karma server.
 -->
-<html>
+<html lang="EN" xml:lang="en">
```

Comment:
`xml:lang` is not correct here as the document is HTML. `xml:lang` is needed when using XHTML.
See: https://www.w3schools.com/tags/ref_language_codes.asp

Setting the `lang` here does not seem right. There is no guarantee that the language is English. Users might very well change their application locale to something else.